matrix/Matrix.hpp

```diff
@@ -270,7 +270,7 @@ class Matrix
     void operator/=(Type scalar)
     {
         Matrix<Type, M, N> &self = *this;
-        self = self * (1.0f / scalar);
+        self = self * (Type(1.0f) / scalar);
```

Comment:
Shouldn't it be `Type(1)` then? In case it's used as double template it doesn't really make sense to start out with a float.

Reply:
I made a follow up for this: https://github.com/PX4/Matrix/pull/96